modules/structs/org_type.go

```diff
@@ -40,6 +40,16 @@ func (vt VisibleType) IsPrivate() bool {
 	return vt == VisibleTypePrivate
 }
 
+// VisibilityMode provides the mode string of the visibility type (public, limited, private)
+func (vt VisibleType) VisibilityMode() string {
```

Comment:
I think `Name` or `String` is a better name for the function.

Reply:
Actually first had it as String before. Will go back to using that.

Reply:
Done